detox/src/devices/allocation/drivers/android/emulator/EmulatorAllocDriver.test.js

```diff
@@ -156,7 +156,13 @@ describe('Allocation driver for Google emulators', () => {
     it('should pre-validate proper AVD configuration', async () => {
       givenValidAVD();
       await allocDriver.allocate(deviceConfig);
-      expect(avdValidator.validate).toHaveBeenCalledWith(avdName);
+      expect(avdValidator.validate).toHaveBeenCalledWith(avdName, deviceConfig.headless);
```

Comment:
Yarik has a good point. Isn't `deviceConfig.headless` is `undefined` here?

Reply:
It should pass without `deviceConfig.headless` arg under the expectation, and I think it's better to be explicit on that:
```suggestion
      expect(avdValidator.validate).toHaveBeenCalledWith(avdName);
```

Reply:
I'm taking the liberty of committing this suggestion 😅